org.eclipse.ice.modeling/src/main/java/org/eclipse/ice/modeling/IWorkflow.java

```diff
@@ -0,0 +1,13 @@
+package org.eclipse.ice.modeling;
+
+import org.eclipse.ice.modeling.workflowEngine.*;
+
+public interface IWorkflow {
+
+	/**
+	 * 
```

Comment:
In general within ICE we say that comments for functions should give some brief description of what the function does

Reply:
Agreed. Classes and interfaces should also have documentation.